saleor/graphql/order/mutations/orders.py

```diff
@@ -560,3 +561,35 @@ def perform_mutation(cls, _root, info, **data):
             cls.get_node_or_error(info, data["id"], only_type=Invoice)
         )
         return super().perform_mutation(_root, info, **data)
+
+
+class UpdateInvoice(ModelMutation):
+    invoice = graphene.Field(Invoice, description="An updated invoice.")
+
+    class Arguments:
+        id = graphene.ID(required=True, description="ID of an invoice to update.")
+        number = graphene.String(description="Invoice number")
+        url = graphene.String(description="URL of an invoice to download.")
```

Comment:
Maby we should wrap this field to `InvoiceUpdateInput`

Reply:
Yes, according to how other model mutations are implemented this one should have the following signature: `(id: ID, input: InvoiceUpdateInput)`.